assets/src/application/ui/layout/entityList/filterBar/EntityListFilterBar.js

```diff
@@ -17,7 +17,9 @@ import './style.css';
  * EntityListFilterBar
  * a group of inputs for controlling how a list of entities is displayed
  */
-const EntityListFilterBar = ({ listId, entityFilters = null }) => {
+const EntityListFilterBar = ({ entityFilters = null, filterState, listId }) => {
+	if (!filterState) return null;
```

Comment:
This is a temp because ticket list now doesn't pass this property; and also as mentioned above this will need to be renamed/ rethinked.